hive/src/main/java/org/elasticsearch/hadoop/hive/pushdown/Utils.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.hadoop.hive.pushdown;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.elasticsearch.hadoop.cfg.Settings;
+import org.elasticsearch.hadoop.hive.EsStoragePredicateHandler;
+import org.elasticsearch.hadoop.util.unit.Booleans;
+
+public class Utils {
+
+    private static final Log log = LogFactory.getLog(Utils.class);
+
+    /**
+     * is pushdown optimization, default true
+     */
+    static String DATA_SOURCE_PUSH_DOWN = "es.internal.hive.pushdown";
+
+    /**
+     * pushdown class path，default org.elasticsearch.hadoop.hive.EsStoragePredicateHandler
+     */
+    static String DATA_SOURCE_PUSH_DOWN_CLASS = "es.internal.hive.pushdown.class";
```

Comment:
Are these actually internal configurations? The `es.internal` namespace is for properties that users are never expected to set themselves, and are reserved for use by the connector code only. If that's not the case for these, I would remove the `internal` part from the config key and prefix it with `ES_`.

If they _are_ meant to be internal properties, then their property names should be prefixed with `ES_INTERNAL` for easier code reading.